packages/runner/injection/cross-origin.js

```diff
@@ -18,7 +23,7 @@ const findCypress = () => {
       if (frame.Cypress) {
         // If the ending $ is included in the template string, it breaks transpilation
         // eslint-disable-next-line no-useless-concat
-        const frameHostRegex = new RegExp(`(^|\\.)${ frame.location.host.replaceAll('.', '\\.') }` + '$')
+        const frameHostRegex = new RegExp(`(^|\\.)${ frame.location.host.replace(/\./gm, '\\.') }` + '$')
```

Comment:
where again was `replaceAll` not defined?

Reply:
Previously es-lint was complaining when i had this defined in cypress.ts (it has since changed there), but now it doesn't complain anymore? I guess i can revert this.